block/manager.go

```diff
@@ -191,14 +200,27 @@ func (m *Manager) Start(ctx context.Context) error {
 		}
 	}
 
-	// Check if the chain is halted
-	err := m.isChainHalted()
-	if err != nil {
-		return err
+	// Check if a proposer on the rollapp is set. In case no proposer is set on the Rollapp, fallback to the hub proposer (If such exists).
+	// No proposer on the rollapp means that at some point there was no available proposer.
+	// In case there is also no proposer on the hub to our current height, it means that the chain is halted.
+	if m.State.GetProposer() == nil {
+		m.logger.Info("No proposer on the rollapp, fallback to the hub proposer, if available")
+		SLProposer, err := m.SLClient.GetProposerAtHeight(int64(m.State.Height()))
```

Comment:
same here, that could fallback to slClient.GetProposer() when not found at specific height (because state info is not submitted yet)

Reply:
otherwise nodes that are higher of the last state update submitted could not start